documentation/docs/pmm-upgrade/migrating_from_pmm_2.md

```diff
@@ -275,34 +283,30 @@ For detailed instructions, see the [Upgrade PMM Client topic](../pmm-upgrade/upg
 ## Step 4: Migrate your API keys to service accounts
 
 PMM 3 replaces API keys with service accounts to enhance security and simplify access management. You can trigger this API key conversion from the UI or from the CLI.
-	
-### From the UI
-PMM automatically migrates existing API keys to service accounts when you first log in as an Admin user. The migration results are displayed in a popup dialog box. 
-
-If no popup appears, it likely means there are no API keys to migrate—this is typical for PMM Servers without connected services.
-	
-### From CLI
-You can also initiate the conversion using the following command. 
-Be sure to replace `admin:admin` with your credentials and update the server address (`localhost` or `127.0.0.1`) and port number (`3000`) if they differ from the defaults:
 
-	
-```sh
-curl -X POST http://localhost:3000/api/serviceaccounts/migrate \
--u admin:admin \
--H "Content-Type: application/json
-```
-	
-The response will display the migration details:
-
-!!! example "Expected output"
+=== "From CLI"
+    You can also initiate the conversion using the following command. 
+    Be sure to replace `admin:admin` with your credentials and update the server address (`localhost` or `127.0.0.1`) and port number (`3000`) if they differ from the defaults:
```

Comment:
> (`localhost` or `127.0.0.1`)

this seems like there are only 2 options localhost or 127.0.0.1.

port 3000 is internal port of grafana. we proxy this on port 8443 which can be mapped to 443 if user used our instruction or script. also we have prefix /graph